sos/report/plugins/saltmaster.py

```diff
@@ -6,20 +6,33 @@
 #
 # See the LICENSE file in the source distribution for further information.
 
-from sos.report.plugins import Plugin, RedHatPlugin, DebianPlugin
+from sos.report.plugins import Plugin, IndependentPlugin
 
 
-class SaltMaster(Plugin, RedHatPlugin, DebianPlugin):
+class SaltMaster(Plugin, IndependentPlugin):
 
     short_desc = 'Salt Master'
 
     plugin_name = 'saltmaster'
     profiles = ('sysmgmt',)
 
-    packages = ('salt-master',)
+    packages = ('salt-master', 'salt-api',)
 
     def setup(self):
-        self.add_copy_spec("/var/log/salt/master")
+        all_logs = self.get_option("all_logs")
```

Comment:
Unnecessary - use `get_option()` in line with your conditionals.

Reply:
Reworked.